pkg/logo/logo.go

```diff
@@ -0,0 +1,11 @@
+package logo
+
+import (
+	_ "embed" // For "go:embed".
+)
+
+//go:embed favicon.svg
+var faviconSvg []byte
+
+// FaviconSvg holds the Buildbarn logo and can be used as favicon in web browsers.
+var FaviconSvg = faviconSvg
```

Comment:
Out of curiosity, why the indirection? We can't make the `go:embed` variable public? I think we should also call this `FaviconSVG`, as in Go they tend to write abbreviations like HTTP, XML, etc. in all caps.